### DISCUSSION TITLE
how to let fold don't display <e4><bd>...

### DISCUSSION THREAD

**Discussion by Freed-Wu** [nvim-treesitter/nvim-treesitter#7046] [Q&A] [answered]
`test.md`:

```markdown
---
title: title
---

你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你你
```

```vim
set foldexpr=nvim_treesitter#foldexpr()
```

![Screenshot from 2024-08-17 17-28-38](https://github.com/user-attachments/assets/b575812f-8e7c-4994-8e6f-bcf8a3ddb67e)

Can we don't display `<e4><bd>...`? Just display entire CJKV character or `...` is OK.

**Comment by clason** [comment, marked as answer]
1. Don't use this function, it's deprecated; use the upstream `vim.treesitter.foldexpr()`.
2. This is related to multibyte unicode handling, which is work in progress in Neovim. Nothing we can do about it here.
